creator-node/src/services/sync/syncQueueService.js

```diff
@@ -0,0 +1,69 @@
+const Bull = require('bull')
```

Comment:
i think the "Service" in the file name is redundant. it's actually clearer as SyncQueue?

Reply:
eh idk, but nbd, changed it